AUTHORS.rst

```diff
@@ -0,0 +1,7 @@
+Authors
+=======
+
+- Predrag Gruevski (github: obi1kenobi)
+- Amartya Shankha Biswas (github: amartyashankha)
+- Jeremy Meulemans (github: jmeulemans)
+- Pedro Mantica (github: pmantica1)
```

Comment:
@bojanserafimov if you want to add yourself here, go for it. Same goes for @qqi0O0, @LWprogramming, and @ReaLNeroM – you have all made substantial contributions to this project, and have earned your spot on this list.

Reply:
Though probably wait until this is merged, it's already a 5000-line PR :)